app/controllers/users.js

```diff
@@ -0,0 +1,15 @@
+const logger = require('../logger');
+const usersService = require('../services/users');
+const signUpValidator = require('../interactors/sign_up_validator');
+
+module.exports.signUp = (req, res, next) => {
+  signUpValidator.validate(req.body);
+
+  usersService
+    .signUp(req.body)
+    .then(() => {
+      logger.info(`Created user: ${JSON.stringify(req.body.firstName)}`);
+      res.status(201).send(req.body.firstName);
```

Comment:
Something I did not notice earlier, it's better to always return a key value pair within an object instead of a plain value. This lets thee user know what that plan value is. For example: send({first_name: req.body.firstName})